### ISSUE TITLE
Support minio zone scale up

### ISSUE THREAD

**Issue by KubrickLiu** [milvus-io/milvus-helm#474]
我想把 milvus-helm 中 minio 的 3 个配置暴露在 values.yaml 里，分别是 drivesPerNode,replicas,zones ，目的是达到扩容的效果，请问下使用过程中扩 zone 能达到 milvus 存储扩容的效果吗？
![image](https://github.com/milvus-io/milvus-helm/assets/24795136/ae655e46-a450-4f30-99c2-19b8e4ff0fd2)

**Comment by LoveEachDay** [comment]
@KubrickLiu Yes you can increase `zones` to add a new server pool which will increase the total capacity for your minio cluster. Beware suppose you have `drivesPerNode` as `1` and `replicas` as `4`, increase `zone` from `1` to `2`, which will add another 4 pods.

**Comment by KubrickLiu** [comment]
I have another question, current minio charts version of milvus helm is too old
![image](https://github.com/milvus-io/milvus-helm/assets/24795136/24b2e564-2eb5-45f3-85a0-fc4371a6d1e2)

Will upgrade to the current version
![image](https://github.com/milvus-io/milvus-helm/assets/24795136/87abe568-d94e-45de-a94a-5aea8dbd7567)
